experimental/CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ All notable changes to experimental packages in this project will be documented
 * fix(sdk-metrics-base): misbehaving aggregation temporality selector tolerance #2958 @legendecas
 * feat(trace-otlp-grpc): configure security with env vars #2827 @svetlanabrennan
 * feat(sdk-metrics-base): async instruments callback timeout #2742 @legendecas
+* feat(node-sdk): add serviceName config option #2867 @naseemkullah
```

Comment:
Sorry for the churn. You probably need to move this changelog line above to the section unreleased.